gender_novels/analysis/analysis.py

```diff
@@ -285,6 +290,161 @@ def instance_dist(novel, word):
     return output
 
 
+def process_medians(helst, shelst, authlst):
+    """
```

Comment:
this needs at documentation and doctests.  Lab policy is: write documentation, then code.